<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinGesturesKCM</class>
 <widget class="QWidget" name="KWinGesturesKCM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="minimumHeight">
      <number>320</number>
     </property>
     <widget class="QWidget">
      <attribute name="title">
       <string>Stroke recorder</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel">
         <property name="text">
          <string>How to use:
1. If using a mouse, place the window so that the record button
is located where you would like to start the stroke
2. Press record
3. Perform the stroke using a mouse or touchpad
4. If using a mouse, wait a short amount of time after stopping
movement
5. Copy the output below and assign it to the stroke property
of a gesture (it is a list)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="recordStroke">
           <property name="icon">
            <iconset theme="media-record"/>
           </property>
           <property name="text">
            <string>Record stroke</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="strokePoints">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copy">
           <property name="icon">
            <iconset theme="edit-copy"/>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTextBrowser" name="aboutText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Failed to load about.html</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
